htdocs/compta/paiement.php

```diff
@@ -760,11 +761,12 @@ function callForResult(imgId)
                     // Print total
                     print '<tr class="liste_total">';
                     print '<td colspan="2" align="left">'.$langs->trans('TotalTTC').'</td>';
-					if (!empty($conf->multicurrency->enabled)) print '<td></td>';
-					if (!empty($conf->multicurrency->enabled)) print '<td></td>';
-					if (!empty($conf->multicurrency->enabled)) print '<td></td>';
-					if (!empty($conf->multicurrency->enabled)) print '<td></td>';
-					if (!empty($conf->multicurrency->enabled)) print '<td align="right" id="multicurrency_result" style="font-weight: bold;"></td>';
+                    if (!empty($conf->multicurrency->enabled)) {
```

Comment:
5 columns before, 4 now.
Is it done on purpose or you forget one column ?

Reply:
@eldy arf ! yes ! thanks